### PR TITLE
Add plugin: Desci

### PR DESCRIPTION
**Pull request by polus-arcticus**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/Obsidian-Desci/Obsidian-Desci

## Release Checklist
- [ ] I have tested the plugin on
  - [ ]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [ ] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
You should move all all code files into a separate `src` folder, that makes it easier to review and maintain.

[// Remember to rename these classes and interfaces!](https://github.com/Obsidian-Desci/Obsidian-Desci/blob/e1867e2ec9f37f955237072640883008cd0be0ad/main.tsx#L19)
Remove this comment

[console.log('clientadd', ExampleClient.address)](https://github.com/Obsidian-Desci/Obsidian-Desci/blob/e1867e2ec9f37f955237072640883008cd0be0ad/main.tsx#L36), and many more.
Please avoid unnecessary logging. If you need it for debugging purposes, add a check to only log during development.

[// This creates an icon in the left ribbon.](https://github.com/Obsidian-Desci/Obsidian-Desci/blob/e1867e2ec9f37f955237072640883008cd0be0ad/main.tsx#L77-L93)
Remove all this sample code.

 [// If the plugin hooks up any global DOM events (on parts of the app that doesn't belong to this plugin)](https://github.com/Obsidian-Desci/Obsidian-Desci/blob/e1867e2ec9f37f955237072640883008cd0be0ad/main.tsx#L96-L104)
Also remove all this code.

[.setName('Use Engine')](https://github.com/Obsidian-Desci/Obsidian-Desci/blob/e1867e2ec9f37f955237072640883008cd0be0ad/settings.ts#L62)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

**Comment by polus-arcticus** [comment]
Sounds good @joethei ill make these changes

**Comment by polus-arcticus** [comment]
Hey @joethei I pushed the changes, updated the tag, and retriggered the ci, :)

File: community-plugins.json [hunk 1]
```diff
@@ -8622,7 +8622,6 @@
     "description": "Open URLs based on a permalink or slug in the note properties. Useful with static site generator such as Jekyll, Hugo, Eleventy, etc.",
     "repo": "kepano/obsidian-permalink-opener"
   },
```
Comment (liamcain) [inline community-plugins.json:8624]:
```suggestion
  },
  {
```

**Pull request merged**